modules/aerodyn/src/AeroDyn_IO.f90

```diff
@@ -1019,6 +1160,38 @@ SUBROUTINE ParsePrimaryFileInfo( PriPath, InitInp, InputFile, RootName, NumBlade
    ! Prevent segfault when no blades specified.  All logic tests on BldNd_NumOuts at present.
    if (InputFileData%BldNd_BladesOut <= 0)   InputFileData%BldNd_NumOuts = 0
 
+
+   !====== Print new and old inputs =====================================================================
+   call printNewOldInputs()
+
+   !====== Advanced Options =============================================================================
+   if ((CurLine) >= size(FileInfo_In%Lines)) RETURN
+
+   call WrScr(' - Reading advanced options for AeroDyn')
+   do CurLine= CurLine, size(FileInfo_In%Lines)
+      sDummy = FileInfo_In%Lines(CurLine)
+      call Conv2UC(sDummy)  ! to uppercase
+      if (index(sDummy, '!') == 1 .or. index(sDummy, '=') == 1 .or. index(sDummy, '#') == 1 .or. index(sDummy, '---') == 1) then
+         ! pass comment lines
+         elseif (index(sDummy, 'SECTAVG')>1) then
+            read(sDummy, '(L1)') InputFileData%SectAvg
+            print*,'   >>> SectAvg        ',InputFileData%SectAvg
```

Comment:
unless this is wrapped in a debugging clause, can we change `print*` to `call WrScr` for compatibility with Simulink?

Reply:
de83885ba